packages/stores/src/queries/concentrated-liquidity/position-by-id.ts

```diff
@@ -135,16 +143,24 @@ export class ObservableQueryLiquidityPositionById extends ObservableChainQuery<L
     return this._canFetch;
   }
 
-  protected setResponse(response: Readonly<QueryResponse<LiquidityPosition>>) {
+  protected setResponse(
+    response: Readonly<QueryResponse<{ position: LiquidityPosition }>>
+  ) {
     super.setResponse(response);
-    this.setRaw(response.data);
+    this.setRaw(response.data.position);
+    const rewardDenoms = Array.from(
+      new Set(
```

Comment:
Good use of set so long as the array is not too big

Reply:
Yes this array should never exceed 10